python/graphscope/gsctl/config.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright 2023 Alibaba Group Holding Limited. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Global configuration"""
+
+import os
+import random
+from string import ascii_letters
+
+import yaml
+
+GS_CONFIG_DEFAULT_LOCATION = os.environ.get(
+    "GSCONFIG", os.path.expanduser("~/.gs/config")
```

Comment:
I remember that we use `~/.graphscope` rather than `~/.gs` before. Keep it consistent would be better.

Reply:
Yes, we do use ~/.graphscope instead of ~/.gs. It is better to maintain consistency.